src/controls/Button.jsx

```diff
@@ -35,60 +36,53 @@ const buttonStyles = {
  * @param {Object} [props.style] Desired JSX-formatted styling object; overrides all other style attributes
  * @returns {import("react").ReactComponentElement}
  */
-export default function Button(props) {
-  let {
-    text,
-    onClick,
-    width,
-    type,
-    size,
-    height,
-    hoverStart,
-    backgroundColor,
-    activeBackgroundColor,
-    style,
-  } = props;
+const Button = (props) => {
+  let { text, onClick, width, type, size, height, style } = props;
 
   // Assign fallback values for styling
   if (!type || !(type in Object.keys(buttonStyles))) {
     type = "default";
   }
 
   return (
-    <HoverBox
-      hoverBackgroundColor={
-        activeBackgroundColor || buttonStyles[type].hoverBackgroundColor
-      }
-      hoverStart={hoverStart || "left"}
+    <AntButton
       style={{
-        alignItems: "center",
         display: "flex",
-        backgroundColor:
-          backgroundColor || buttonStyles[type].standardBackgroundColor,
-        color: "white",
+        borderRadius: "0",
+        backgroundColor: colors.TEAL_ACCENT,
+        textTransform: "uppercase",
+        width: width || "min(300px, 70vw)",
+        height: height || "auto",
         padding: 15,
         paddingLeft: 30,
         paddingRight: 30,
         fontSize: 15,
         fontFamily: "Roboto",
         fontWeight: 500,
         letterSpacing: 2.4,
-        cursor: "pointer",
-        textTransform: "uppercase",
-        width: width || "min(300px, 70vw)",
-        height: height,
-        justifyContent: "center",
         textAlign: "center",
-        outline: "none",
-        boxShadow: "none",
-        userSelect: "none",
+        justifyContent: "center",
+        color: "#ffffff",
         ...style,
       }}
+      onMouseOver={(e) =>
+        (e.currentTarget.style.backgroundColor =
+          buttonStyles[type].hoverBackgroundColor) &&
+        (e.currentTarget.style.borderColor =
+          buttonStyles[type].hoverBackgroundColor)
+      }
+      onMouseOut={(e) =>
+        (e.currentTarget.style.backgroundColor =
+          buttonStyles[type].standardBackgroundColor) &&
+        (e.currentTarget.style.borderColor =
+          buttonStyles[type].standardBackgroundColor)
+      }
       size={size ? size : width ? `${width}px` : "300px"}
       onClick={onClick}
-      aria-label={text}
     >
-      {text}
-    </HoverBox>
+      {text ? text : `🎉New Button`}
```

Comment:
```suggestion
      {text}
```
In this case, I actually think we're fine with the app breaking, since a button with placeholder text is as bad for the UX as a broken button, in my opinion